### ISSUE TITLE
All GET vars are converted to lowercase

### ISSUE THREAD

**Issue by adoerler** [koninklijke-collective/my_redirects#49]
Hi,

if one sets a redirect containing parameter values in uppercase, e.g. a Google Campaign like:
`?utm_medium=Èmail&utm_campaign=TEST-Campaign`
the values (and parameter names) get converted into lowercase which leads in the case of Google to two different campaign sources.
I'm sure there are other cases where capital letters in parameters or values must be respected as well.

I didn't find an option to control this behavior and in the code it looks like `toLower` is one of the first actions applied to the redirect path.

What was your intention to do a conversion to lower case anytime and is there a workaround to keep capital letters.

Kind regards and thank you for this great extension
Andreas

**Comment by mbrodala** [comment]
I agree that this should not be done unconditionally but it's still nice to have, e.g. with a case-sensitivity flag per redirect.

**Comment by pkracht** [comment]
Also agree and need this fix as soon as possible... see #60 
The URL can be camelcase or whatever containing a upper-lower-case token. For example adword and analytics makers (see https://support.google.com/analytics/answer/2938246?hl=en)
